src/noc/noc.routes.ts

```diff
@@ -13,7 +13,7 @@ export const ROUTES: Routes = [
       {
         path: '',
         component: null,
-        canActivate: [ HealthCheckGuard, NocGuard ],
+        canActivate: [ HealthCheckGuard, FeatureToggleGuard ],
```

Comment:
NocGuard removed?

Reply:
Yes @udaydenduluri33, @ronaldmansveld removed it when he was doing the development for refactoring header EUI-4575.